<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditChargesWidget</class>
 <widget class="QWidget" name="EditChargesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="qhlay_lowChargeButtons">
     <item>
      <widget class="QLabel" name="ql_ajdLowCharges">
       <property name="text">
        <string>Adjust low charges</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ReturnAcceptableTableView" name="qtblView_charges"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="qhlay_highChargeButtons">
     <item>
      <widget class="QLabel" name="ql_adjHighCharges">
       <property name="text">
        <string>Adjust high charges</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReturnAcceptableTableView</class>
   <extends>QTableView</extends>
   <header>src/ui/returnacceptabletableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
